acapy/plugins/firebase_push_notifications/firebase_push_notifications/v1_0/manager.py

```diff
@@ -0,0 +1,106 @@
+from datetime import timedelta
+from dateutil import parser
+import logging
+import json
+import os
+import requests
+
+import google.auth.transport.requests
+from google.oauth2 import service_account
+
+from aries_cloudagent.messaging.util import time_now, datetime_now
+from .models import FirebaseConnectionRecord
+
+from .constants import SCOPES, BASE_URL, ENDPOINT_PREFIX, ENDPOINT_SUFFIX
+
+PROJECT_ID = os.environ.get('FCM_PROJECT_ID')
+FCM_ENDPOINT = ENDPOINT_PREFIX + PROJECT_ID + ENDPOINT_SUFFIX
+FCM_URL = BASE_URL + '/' + FCM_ENDPOINT
+MAX_SEND_RATE_MINUTES = int(os.environ.get('FCM_MAX_SEND_RATE_MINUTES'))
```

Comment:
Would it make sense to provide a reasonable default value here, rather than requiring a user-defined one?

Reply:
@jamshale could we add a reasonable default (`5`minutes maybe?) here so that if the global setting is missing the plugin will still work? Any reason NOT to do this?